src/components/GestureButtons.tsx

```diff
@@ -88,25 +131,25 @@ const btnStyles = StyleSheet.create({
   },
 });
 
-export class RectButton extends React.Component {
-  static propTypes = BaseButton.propTypes;
-
+export class RectButton extends React.Component<RectButtonProperties> {
   static defaultProps = {
     activeOpacity: 0.105,
     underlayColor: 'black',
   };
 
-  constructor(props) {
+  constructor(props: RectButtonProperties) {
     super(props);
-    this._opacity = new Animated.Value(0);
+    this.opacity = new Animated.Value(0);
   }
 
-  _onActiveStateChange = (active) => {
+  private opacity: Animated.Value;
+
+  private onActiveStateChange = (active: boolean) => {
     if (Platform.OS !== 'android') {
-      this._opacity.setValue(active ? this.props.activeOpacity : 0);
+      this.opacity.setValue(active ? this.props.activeOpacity! : 0);
```

Comment:
Not null type assertion.

Reply:
It is inevitable in class component with `defaultProps`, which take care of always providing the value.